core/src/main/java/org/jruby/RubyRandom.java

```diff
@@ -680,6 +680,12 @@ public IRubyObject marshal_load(ThreadContext context, IRubyObject arg) {
         return this;
     }
 
+    // c: random_s_bytes
+    @JRubyMethod(meta = true)
+    public static IRubyObject bytes(ThreadContext context, IRubyObject recv, IRubyObject arg) {
+        return ((RubyRandom) (context.runtime.getRandomClass()).getConstant("DEFAULT")).bytes(context, arg);
```

Comment:
MRI works even if you change the `Random::DEFAULT` constant
so we should follow that, use `context.runtime.getDefaultRand()` to retrieve the internal (default) random.
the existing bytes should be refactored to an internal static helper that receives a `RandomType random` arg

Reply:
@kares i will try refactoring.

Reply:
@kares Please review. Thank you.